Tiltfile

```
@@ -10,6 +10,14 @@ cmd_button(
     text='make codegen-local',
 )
 
+cmd_button(
+    'make test-local',
```

```suggestion
    test',
```

I'd suggest having a shorter name instead.